tools/ide_plugins/pysa-vscode/tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+	"compilerOptions": {
+		"module": "commonjs",
+		"target": "es6",
+		"outDir": "out",
+		"lib": [
+			"es6"
+		],
+		"sourceMap": true,
+		"rootDir": "src",
+		"strict": true   /* enable all strict type-checking options */
+		/* Additional Checks */
+		// "noImplicitReturns": true, /* Report error when not all code paths in function return a value. */
```

Comment:
Why disable these? Looks like they're all enabled (plus the additional `noImplicitAny`) in the pyre plugin:
https://github.com/facebook/pyre-check/blob/master/tools/ide_plugins/vscode/tsconfig.json